build/rules.mk

```
@@ -155,10 +155,12 @@ define CONTAINER_RULE
 .$(BUILDSTAMP_NAME)-container: bin/$(ARCH)/$(BINARY)
 	@echo "container: bin/$(ARCH)/$(BINARY) ($(CONTAINER_NAME))"
 	@docker pull $(BASEIMAGE)
-	@docker build					\
+	@docker buildx	build				\
```

```suggestion
	@docker buildx build				\
```

nit, it seems you have a tab instead of  a space, or was that on purpose?